index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta char="utf-8">
+	<title>F to C | C to F</title>
+	<link rel="stylesheet" type="text/css" href="../css/main.css">
+</head>
+<!-- In the HTML, have one input field where someone can enter in a temperature. -->
+	<body>
```

Comment:
Here, add an id to this body tag that you can reference in your document.querySelector statement in your .js file.